### DISCUSSION TITLE
session timeout

### DISCUSSION THREAD

**Discussion by Klauss214659** [AdguardTeam/AdGuardHome#5971] [Ideas]
Can be possible to change the timeout session time from "session_ttl: 720h" (30 days) to 15 minutes for example "session_ttl: 15m" or "session_ttl: 0.25h" or any other custom value.

Thanks.

**Comment by ainar-g** [comment]
Have you tried `session_ttl: 15m`?  Because it should just work, from what I can tell.

**Comment by Klauss214659** [comment]
Cool, i know nothing about yaml syntax that's why i asked, thank you for your help.
